src/components/DatePicker.tsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from 'react';
+import { format } from 'date-fns';
+import { CalendarIcon } from 'lucide-react';
+import { cn } from '@/lib/utils';
+import { Button } from '@/components/ui/button';
+import { Calendar } from '@/components/ui/calendar';
+import {
+    Popover,
+    PopoverContent,
+    PopoverTrigger,
+} from '@/components/ui/popover';
+
+interface DatePickerProps {
+    selectedDate: Date | undefined;
+    onSelect: (date: Date | undefined) => void;
+}
+
+export function DatePicker({ selectedDate, onSelect }: DatePickerProps) {
+    const [isDatePickerOpen, setIsDatePickerOpen] = useState(false);
+
+    return (
+        <Popover open={isDatePickerOpen} onOpenChange={setIsDatePickerOpen}>
+            <PopoverTrigger asChild>
+                <Button
+                    variant={'outline'}
```

Comment:
You can also do this
```suggestion
                    variant="outline" 
```

Reply:
Fixed it.
https://github.com/dhruvdhaduk-simform/todo/pull/4/commits/381b5077414db864515787b0df5dc2cc64451be8